src/features/user/BulkCodes/BulkCodes.module.scss

```diff
@@ -0,0 +1,112 @@
+@import '../../../theme/theme';
+// @import './themeCopy';
+
+/* .contentContainer {
```

Comment:
remove commented code if not needed.

Reply:
Removed in [fix: update donation success message](https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1468/commits/d71ca26d409b76e247fa9d9079e60869e9cdfb67)